README.md

```diff
@@ -126,48 +134,59 @@ export const myClient = GqlClient(
 ```js
 import { myClient } from './clients'
 
-myClient.query(GQL_QUERY) // => Promise<{ data?: Query, errors?: any[] }>
+myClient
+  .query(GQL_QUERY) // => Promise<{ data?: Query, errors?: any[] }>
   .then(console.log)
-  
-myClient.mutation(GQL_MUTATION) // => Promise<{ data?: Mutation, errors?: any[] }>
+
+myClient
+  .mutation(GQL_MUTATION) // => Promise<{ data?: Mutation, errors?: any[] }>
   .then(console.log)
 
-myClient.subscription(GQL_SUBSCRIPTION) // => Observable<{ data?: Subscription, errors?: any[] }>
+myClient
+  .subscription(GQL_SUBSCRIPTION) // => Observable<{ data?: Subscription, errors?: any[] }>
   .subscribe({
     next: console.log
   })
 ```
+
 `GQL_QUERY`/`GQL_MUTATION`/`GQL_SUBSCRIPTION` is the object that will be converted into GraphQL request.
 Here is an example, to showcase the format:
+
 ```js
 myClient.query({
   // object field with arguments
-  user: [{ id: 'USER_ID' }, {
-    // scalar field
-    username: 1,
-    email: 1,
-    // scalar field with arguments
-    wasEmployed: [{ recently: true }],
-    // object field without arguments
-    friends: {
+  user: [
+    { id: 'USER_ID' },
```

Comment:
the issue is this (IIRC, prettier does not allow to turn this behavior off)

it's very important for queries specifically since they're meant to be looking visually similar to graphql
```gql
query {
  user(id: "USER_ID") {
    username
    email
  }
}
```
looks like
```ts
query({
  user: [{ id: 'USER_ID' }, {
    username: 1,
    email: 1,
  }],
})
```
but not
```ts
query({
  user: [
    { id: 'USER_ID' },
    {
      username: 1,
      email: 1,
    },
  ],
})
```

Reply:
Although the answer to this issue could also be "don't run prettier on README" 🙂

Reply:
so you expect the users of this library to not use prettier? :sweat_smile: 
IMHO it is just an object-not a graphQL query. GraphQL queries are nice, I personally like writing them, but this is not a Graphql query. Shuffling `\n` around doesn't really magically make it into one.

Reply:
I can add markdown files to .prettierignore if you prefer formatting them manually. My preference is to let prettier do it, but it is your repo.

Reply:
lets' do that + prettify REAMDE as a one-time operation, but then manually change query examples back to original array formatting

Reply:
should be a good compromise, otherwise I very much like prettier's philosophy